README.md

```diff
@@ -124,7 +124,8 @@ users, so let's start with that.
 
 ```
 mkdir -p $ZSH_CUSTOM/plugins/tldr
-ln -s bin/completion/zsh/_tldr $ZSH_CUSTOM/plugins/tldr/_tldr
```

Comment:
This path assumes you have a local clone of the codebase. But maybe that wasn't clear. I'd like to keep both versions of this. Maybe we can clarify and say that if you have a local clone of the repo, use `bin/completion/zsh/_tldr` or else use `$(npm config get prefix)/lib/node_modules/tldr/bin/completion/zsh/_tldr` once you have installed the tool?